src/bridge/cockpitinternalmetrics.c

```diff
@@ -242,6 +240,11 @@ cockpit_internal_metrics_sample (CockpitSamples *samples,
 {
   CockpitInternalMetrics *self = COCKPIT_INTERNAL_METRICS (samples);
 
+  if (self->omit_instances)
```

Comment:
I think the for loop needs braces too. Generally any block that is more than one line should have them.